### PR TITLE
Add step in RELEASE.md to check neo-express version

### PR DESCRIPTION
**Pull request by mialbu**

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -5,6 +5,12 @@ at [neow3j.io](https://neow3j.io) is up-to-date.
 
 ## Prepare for Source Code Release
 
+> If this release considers a Neo version update (e.g. `3.4.0` &rarr; `3.5.0`): Check whether the [Neo-Express
+> version](https://github.com/neo-project/neo-express/releases) used in the resources of neow3j's `test-tools` module
```
Comment (gsmachado) [inline RELEASE.md:9]: Can you mention, somewhere, the following file?

`test-tools/src/main/resources/test.properties`, specifically the `neoExpressDockerImage` property

File: RELEASE.md [hunk 2]
```diff
@@ -5,6 +5,12 @@ at [neow3j.io](https://neow3j.io) is up-to-date.
 
 ## Prepare for Source Code Release
 
+> If this release considers a Neo version update (e.g. `3.4.0` &rarr; `3.5.0`): Check whether the [Neo-Express
```
Comment (gsmachado) [inline RELEASE.md:8]: maybe., add it as a "major" Neo version update?

**Review by gsmachado — COMMENTED** [review]
Check my comments. Minor.

File: RELEASE.md (hunk 2 above)
Comment (mialbu) [reply RELEASE.md]: yes, added in e79a774

File: RELEASE.md (hunk 1 above)
Comment (mialbu) [reply RELEASE.md]: yes, added in e79a774

**Review by gsmachado — APPROVED** [review]
Fine for me. 👍 

Go go go!

**Pull request merged**